audio-player.js

```diff
@@ -84,40 +84,40 @@ module.exports = class AudioPlayer {
       throw new Error(
         'Some form of audio playback capability is required');
     }
+    if (this.canUseAudio) {
```

Comment:
`!(this.canUseAudio || this.canUseCordovaMedia)`

Will only pass if both are false.

`canUseAudio` can still be true, but `canUseCordovaMedia` does not need to. 

It's a bit convoluted though I made it better now.